anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/rcf/ModelParameters.java

```diff
@@ -0,0 +1,21 @@
+package com.expedia.adaptivealerting.anomdetect.rcf;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+/**
+ * @author Tatjana Kamenov
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+public class ModelParameters {
```

Comment:
To be consistent with the other models should the be RcfParams e.g. https://github.com/ExpediaDotCom/adaptive-alerting/blob/a761ad93d51f2ab8f7ba0e98359a5eb1bc0c14b2/anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/ewma/EwmaParams.java

Reply:
These are optional after-training parameters specific to rcf so I did not want to make confusion.